python/tests/unit/arcticdb/version_store/test_resample.py

```diff
@@ -503,4 +505,94 @@ def test_resampling_empty_type_column(lmdb_version_store_empty_types_v1):
     q = QueryBuilder()
     q = q.resample("s").agg({"col": "first"})
     with pytest.raises(SchemaException):
-        lib.read(sym, query_builder=q)
\ No newline at end of file
+        lib.read(sym, query_builder=q)
+
+class TestResamplingOffset:
+
+    @staticmethod
+    def all_aggregations_dict(col):
+        return {f"to_{agg}": (col, agg) for agg in ALL_AGGREGATIONS}
+
+    @pytest.mark.parametrize("closed", ["left", "right"])
+    def test_offset_smaller_than_freq(self, lmdb_library, closed):
```

Comment:
Why not `lmdb_version_store_v1`? (for all test cases)

Reply:
No particular reason, `lmdb_version_store_v1` is library V1 `lmdb_library` is V2. Should I change it to `lmdb_version_store_v1` to match the other tests?

Reply:
Yeh, unless there is a good reason not to, `lmdb_version_store_v1 ` should be the default